packages/config/src/bridges/polygonpos.ts

```diff
@@ -136,24 +136,21 @@ export const polygonpos: Bridge = {
     isIncomplete: true,
     addresses: [
       {
-        address: '0x40ec5B33f54e0E8A33A975908C5BA1c14e5BbbDf',
```

Comment:
Is the change in contracts order an intended change? This will be reflected on the frontend

Reply:
Yes, it was consulted with @antooni